src/unix/os390.c

```diff
@@ -118,9 +119,10 @@ void uv__platform_loop_delete(uv_loop_t* loop) {
 
 
 uint64_t uv__hrtime(uv_clocktype_t type) {
-  struct timeval time;
-  gettimeofday(&time, NULL);
-  return (uint64_t) time.tv_sec * 1e9 + time.tv_usec * 1e3;
+  uint64_t timestamp;
+  __stckf((unsigned long long*)(&timestamp));
```

Comment:
Is the cast actually necessary?  I mean, isn't `uint64_t` already an `unsigned long long`?  It's either superfluous or dangerous.